tests/compressibility_iterated_stokes.prm

```diff
@@ -108,11 +108,11 @@ subsection Boundary velocity model
 end
 
 subsection Postprocess
-  set List of postprocessors = compressibility
+  set List of postprocessors = compressibility, visualization
 
   subsection Visualization
 
-    set List of output variables      = density
+    set List of output variables      = density, volumetric strain rate
```

Comment:
just copy/clone the test. we like having tests that test one particular aspect, rather than many things at the same time